src/main/js/main_html5.js

```diff
@@ -193,6 +193,7 @@ require("../../../html5-common/js/utils/environment.js");
     var playQueued = false;
     var isSeeking = false;
     var currentTime = 0;
+    var currentVolumeSet = 0;
```

Comment:
Does this value need to be reset when the stream changes?

Reply:
I don't believe so, the player maintains volume settings when stream changes, right? Please correct me if I'm wrong.

Reply:
It "should".

Reply:
Oh you're right. I'm used to resetting all of these vars, but it wouldn't apply for volume.